src/Scenario/AbstractGroupScenario.php

```diff
@@ -27,7 +27,7 @@ public function setDeepLink(string $deepLink): void
 
     public function __construct(?Proxy $proxy = null, ?ClientGeneratorInterface $generator = null)
     {
-        parent::__construct($proxy, $generator);
+        parent::__construct($generator, $generator, $proxy);
```

Comment:
Two generators passing, is it ok?

Reply:
ok, the second one is not really used in this case